addons/l10n_ar/models/account_move.py

```diff
@@ -34,6 +34,17 @@ def _l10n_ar_get_document_number_parts(self, document_number, document_type_code
     l10n_ar_afip_service_start = fields.Date(string='AFIP Service Start Date', readonly=True, states={'draft': [('readonly', False)]})
     l10n_ar_afip_service_end = fields.Date(string='AFIP Service End Date', readonly=True, states={'draft': [('readonly', False)]})
 
+    def _is_manual_document_number(self):
+        res = super()._is_manual_document_number()
+        # NOTE: There is a corner case where 2 sales documents can have the same number for the same DOC from a different vendor,
+        # in that case, the user can create a new Sales Liquido Producto Journal
+
+        # When issuer is supplie the numbering works opposite (supplier enumerate the invoices, and the customer encode the bill)
+        if self.country_code == 'AR' and self.l10n_latam_use_documents and ((self.journal_id.type == 'sale' and not self.journal_id.l10n_ar_is_pos) or (
```

Comment:
@zaoral update the condition here if POS always electronic

Reply:
Hi @vbe-odoo 
I have applied the changes but not sure if they are the right changes. I understood/applied the changes taking into account that is it a Pos System, do not take into account if it is electronic or not. Please let me know if you think we should take into account if it is electronic.
Wait your feedback

Best Regards

Reply:
Hi @zaoral !
Dismiss my message about the electronic hehe! If l10n_ar_is_pos = True (selected) then it should work as a regular invoice (taking the number from AFIP and not being manual)
Cheers!

Reply:
So it should work now, as you took into account the POS.
I can test! 🚀

Reply:
Hello @zaoral can you help me do a rebase on this PR?
As there is an issue that my be solved with a rebase or if not if you can help me check what's happening.
So, when installing AR there is a conflict with some demos data DOCs and then it does not install the demo data, see the screenshot below:
<img width="995" alt="image" src="https://user-images.githubusercontent.com/45669815/215124990-c35ef204-8369-4dc1-8727-bf189e2b05ee.png">

Thanks!